js/client.js

```diff
@@ -67,8 +67,7 @@ export const ClientAPI = superclass =>
          */
         async deleteClient(name) {
             const url = this.baseUrl + `clients/${name}`;
-            const client = await this.delete(url);
-            return client;
+            return await this.delete(url);

```

Comment:
I know this is dumb (returning nothing) but keep it as it is otherwise we'll have to fix this in all JS API clients, which is a bummer.